low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -30,6 +30,7 @@
 from framework.utils.store_factory import store
 from framework.utils import encryptor
 from framework.base.sspl_constants import ServiceTypes, COMMON_CONFIGS
+from framework.utils.conf_utils import *
```

Comment:
import * usually not recommeded, import only whats needed per file

Reply:
Had already corrected as part of codacy issue fixes. Available in new changes.